book/graphs/other/ltg01.qmd

```diff
@@ -187,12 +194,13 @@ g52 <- g52 + stat_summary(
   fun.data = stat_median_ci, geom = "errorbar",
   aes(group = 1, linetype = median_text)
 )
-g52 + guides(linetype = guide_legend(title = "Aggregate"))
+plot <- g52 + guides(linetype = guide_legend(title = "Aggregate"))
+plot
 ```
 
 #### Changing Confidence Level
 
-```{r 2_2_change_conf_level}
+```{r plot53, test = list(plot_v53 = "plot")}
 # Change the confidence level of interval.
 # Note: check `?stat_mean_ci()` and `?stat_median_ci()` for further fine tuning.
 median_text <- "Median +/- 80% CI"
```

Comment:
Snapshot fails because the `.rds` file cannot access `median_text` which is internally defined.
```
! could not find function "median_text"
```